notificationworkerlambda/harvester-cfn.yaml

```diff
@@ -0,0 +1,230 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Consumes events produced by the notification service and batches notifications for senders to send
```

Comment:
Consumes events produced by the notification service and fetches batches of tokens for senders to send